docs/_template.html

```diff
@@ -94,9 +89,7 @@
     src="https://cdnjs.cloudflare.com/ajax/libs/JavaScript-autoComplete/1.0.4/auto-complete.min.js"></script>
   <script type="text/javascript" src="{{root}}content/fsdocs-search.js"></script>
   <!-- END SEARCH BOX: this adds support for the search box -->
-  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
-    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
-    crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-A3rJD856KowSb7dwlZdYEkO39Gagi7vIsF0jrRAoQmDKKtQBHUuLZ9AsSv4jD4Xa" crossorigin="anonymous"></script>
```

Comment:
If you really want to bump bootstrap, you would also need to update the local files at https://github.com/fsprojects/fantomas/blob/8593cb20c867699bedc6a9c5c735fa4532db27ed/docs/.style/bootstrap/scss/bootstrap.scss.

I'm ok with updating this version of bootstrap, although you typically won't update this without any real reason? Do you have one in this case?